exercises/bank.js

```diff
@@ -1,25 +1,51 @@
 
 
 class Bank{
-	constructor(){
+	constructor(bank_name){
+		this.bank_name = bank_name;
+		this.all_accounts = {};
 		//takes in the name of the bank.
 		//makes an object to store all accounts into
 	}
-	makeAccount( ){
+	makeAccount(accountNum ){
+		if(this.all_accounts[accountNum]){
+			return false;
+		}
+		else {
+			this.all_accounts[accountNum] = new Account ();
+			return this.all_accounts[accountNum];
+		}
 		//makes an account
 		//takes in the account number of an account
 		//checks if the account number already exists in the account storage object in the constructor
 			//if already exists, returns false
 		//otherwise makes an new Account object, storing it into the storage object in the constructor
 		//returns the Account object that was constructed
 	}
-	checkForAccount(  ){
+	checkForAccount( accountNum ){
+		if(this.all_accounts[accountNum]){
```

Comment:
if you used hasOwnProperty, it returns true or false so you can simplify this code further.